ledger-tool/src/main.rs

```diff
@@ -4330,6 +4350,78 @@ fn main() {
                     eprintln!("{}", err);
                 }
             }
+            ("new_ads_file", Some(arg_matches)) => {
```

Comment:
@jeffwashington: FYI:  this is the command that creates the new accounts-data-storage from from an append_vec file and verify its correctness.